### ISSUE TITLE
Request feature: Strip XMLNS data

### ISSUE THREAD

**Issue by b1gerhardt** [nashwaan/xml-js#40]
I have XML files that are exported from Excel and have XMLNS prefix data. I don't need to use the data and it's awkward to use objects created with colons in the key.

The fix that meets my needs is simply to add this line at line 140 of xml2js.js:
    name = name.slice ( name.search ( ':' ) + 1 ) ;

I suspect a full implementation would be more work and I'm happy to do that and propose a check-in if you'd like.

**Comment by nashwaan** [comment]
Thanks Barry for your suggestion. I am working now to provide a generalized solution for this kind of issue. Just bear with me for few days.

**Comment by nashwaan** [comment]
Hi @b1gerhardt , just published v1.6.0 of this library which now supports adding [custom functions](https://github.com/nashwaan/xml-js#options-for-custom-processing-functions) for additional processing.

In your case, you can modify element name by providing this option:

```js
convert.xml2js(xml, {compact: false, elementNameFn: function(name) {
    return name.slice(name.search(':') + 1);
}})
```

If this did not work, please provide an example of what you are trying to do.

**Comment by b1gerhardt** [comment]
Awesome. Thanks!
